prism_compile.c

```diff
@@ -2118,6 +2118,8 @@ pm_compile_node(rb_iseq_t *iseq, const pm_node_t *node, LINK_ANCHOR *const ret,
       case PM_NIL_NODE:
         PM_PUTNIL_UNLESS_POPPED
         return;
+      case PM_NO_KEYWORDS_PARAMETER_NODE:
+        return;
```

Comment:
I don't think we can return here immediately without marking the iseq as not accepting keyword parameters. I'm seeing this for CRuby right now:

```
irb(main):003> RubyVM::AbstractSyntaxTree.parse("def foo(**nil); end")
=> 
(SCOPE@1:0-1:19
 tbl: []
 args: nil
 body:
   (DEFN@1:0-1:19
    mid: :foo
    body:
      (SCOPE@1:0-1:19
       tbl: []
       args:
         (ARGS@1:8-1:13
          pre_num: 0
          pre_init: nil
          opt: nil
          first_post: nil
          post_num: 0
          post_init: nil
          rest: nil
          kw: false
          kwrest: false
          block: nil)
       body: nil)))
irb(main):004> RubyVM::AbstractSyntaxTree.parse("def foo; end")
=> 
(SCOPE@1:0-1:12
 tbl: []
 args: nil
 body:
   (DEFN@1:0-1:12
    mid: :foo
    body:
      (SCOPE@1:0-1:12
       tbl: []
       args:
         (ARGS@1:7-1:7
          pre_num: 0
          pre_init: nil
          opt: nil
          first_post: nil
          post_num: 0
          post_init: nil
          rest: nil
          kw: nil
          kwrest: nil
          block: nil)
       body: nil)))
```

looks like `kw` gets set to `false` if this parameter is present.